src/styles/header/header.scss

```diff
@@ -0,0 +1,68 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 1200px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  margin: 0 auto;
+  box-sizing: border-box;
+  font-size: 12px;
+}
+
+@media screen and (max-width: 1024px) {
+  .header__container {
+    max-width: 1024px;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    text-transform: uppercase;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    position: relative;
+    display: inline-block;
+    height: 60px;
+    line-height: 60px;
+    color: $color-font;
+    font-weight: 500;
+    text-decoration: none;
+    cursor: pointer;
+
+    transition: all 300ms ease-in-out;
+
+    &:hover {
+      color: $main-color;
+    }
+  }
+}
+
+.logo__img {
```

Comment:
It is better to use a more descriptive class name for the logo image, such as .header__logo-img, to follow the BEM naming convention.